nvidia-driver-toolkit/entrypoint.sh

```diff
@@ -29,6 +29,16 @@ else
     echo "No DRIVER_LOCATION specified. skipping..."
 fi
 
+# update nvidia driver firmware search path
+# /run/nvidia/driver/lib/firmware is mount under /lib/firmware in the container 
+# by the daemonset manifests
+EXISTS=$(grep '/run/nvidia/driver/lib/firmware' /sys/module/firmware_class/parameters/path)
+if [ -z "${EXISTS}" ]
+then
+    echo "Adding firmware search path to /sys/module/firmware_class/parameters/path"
+    echo  '/run/nvidia/driver/lib/firmware' >> /sys/module/firmware_class/parameters/path
```

Comment:
This could just be `>` not `>>` (I did some quick testing and it looks like /sys/module/firmware_class/parameters/path will only ever take a single line)